src/core_ocean/shared/mpas_ocn_vmix.F

```diff
@@ -663,7 +830,8 @@ subroutine ocn_vmix_init(domain, err)!{{{
       logical, pointer :: config_disable_vel_vmix, config_disable_tr_vmix
       logical, pointer :: config_use_implicit_bottom_drag
       real (kind=RKIND), pointer :: config_implicit_bottom_drag_coeff
-      logical, pointer :: config_Rayleigh_friction, config_Rayleigh_bottom_friction
+      logical, pointer :: config_Rayleigh_friction, config_Rayleigh_bottom_friction, &
+                          config_Rayleigh_damping_coeff_depth_variable
```

Comment:
@pwolfram There is a mismatch here. It is supposed to be `config_Rayleigh_damping_depth_variable`, right? You have coeff in there by mistake.  I merged coastal into ocean/develop locally, this breaks testing using debug.

I can fix it. Just checking that you meant this variable.

Reply:
@pwolfram I ran out of time. If you get a chance, please correct config_Rayleigh_damping_coeff_depth_variable to config_Rayleigh_damping_depth_variable. If you don't have time, we can do this later.